pkg/subnet/public.go

```diff
@@ -94,7 +94,7 @@ func (d *PublicDeployer) AddValidator(
 		if err != nil {
 			return false, nil, err
 		}
-		ux.Logger.PrintToUser("Transaction successful, transaction ID: %s", id)
+		ux.Logger.PrintToUser("Transaction submitted, transaction ID: %s", id)
```

Comment:
If it is already finalized, we can say successful, I just thought it was only submitted